app/build.gradle

```
@@ -38,6 +38,10 @@ android {
 }
 
 dependencies {
+    // Glide
+    implementation 'com.github.bumptech.glide:glide:4.13.0'
+    annotationProcessor 'com.github.bumptech.glide:compiler:4.13.0'
```

`annotationProcessor`의 경우 자바식 어노테이션 프로세서입니당..!

```suggestion
    kapt 'com.github.bumptech.glide:compiler:4.13.0'
```

로 바꿔주시면 좋을 것 같아요 :)

---

오오... 처음 알았네요..!
근데 coli로 수정해서 삭제했습니다.. 😅